.circleci/config.yml

```diff
@@ -1,94 +1,46 @@
-version: 2
+version: 2.1
+
+orbs:
+  queue: eddiewebb/queue@volatile
+
+workflows:
+  build_and_publish:
+    jobs:
+      - queue/block_workflow:
+          context: oneflow
+          only-on-branch: master
+          time: "10" # minutes max wait
+      - build:
+          context: oneflow
+          requires:
+            - queue/block_workflow
+
 jobs:
-  checkout_and_test:
+  build:
     docker:
-      - image: circleci/node:8-jessie
+      - image: circleci/node:12-stretch
     steps:
       - checkout
       - restore_cache:
           keys:
-            - v1-npm-deps-{{ .Branch }}-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
-            - v1-npm-deps-{{ .Branch }}-{{ checksum "package.json" }}
-            - v1-npm-deps-{{ .Branch }}
-            - v1-npm-deps-
+            - v2-npm-deps-{{ checksum "package.json" }}-{{ checksum "package-lock.json" }}
+            - v2-npm-deps-{{ checksum "package.json" }}
+            - v2-npm-deps-
       - run:
           name: install NPM dependencies
           command: |
-            echo -e "//registry.npmjs.org/:_authToken=$NPM_TOKEN\nscope=@oneflow" > .npmrc
-            npm install
+            echo -e "registry=https://npm.pkg.github.com/oneflow\n//npm.pkg.github.com/:_authToken=$GH_TOKEN\nscope=@oneflow" > .npmrc
```

Comment:
We want to continue publishing to `npmjs.com` for this package, since it's a public package, so this line should stay the same as the original

Reply:
fixed